ori/bucket/server/bucket_list.go

```diff
@@ -149,6 +149,10 @@ func (s *Server) getBucketForID(ctx context.Context, id string) (*oriv1alpha1.Bu
 }
 
 func (s *Server) ListBuckets(ctx context.Context, req *oriv1alpha1.ListBucketsRequest) (*oriv1alpha1.ListBucketsResponse, error) {
+	log := s.loggerFrom(ctx)
+
+	log.V(1).Info("Listing buckets", "args", *req)
```

Comment:
I would propose to decrease the log level, if we would like to keep this log line. I think this information is not needed on every call of the ListBuckets endpoint.

Reply:
Done